### PR TITLE
refactor(notification): revised notification and notifications compon…

### PR DESCRIPTION
**Pull request by Erykahh1982**
Dear Barni and Eszter,

Please revise my modifiactions made on components Notification and Notifiactions --> Now it has parent-child relationship implemented.

Thank you
Erika

### CODE REVIEW

File: Tribes/src/app/notification/notification.component.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    border-radius: 20px;
+    padding-top: 10px;
+    color: whitesmoke;
+    list-style-type: none;
+}
+
+img{
+    width: 45px;
+    padding: 5px;
+}
```
Comment (etterke) [inline Tribes/src/app/notification/notification.component.css]: It says no new line at the end of code. Did you run it past linter? Just asking, you may have...

File: Tribes/src/app/notification/notification.component.html [hunk 2]
```diff
@@ -1 +1,3 @@
-<p>Notification works!</p>
\ No newline at end of file
+<div class="container-fluid">
+    <img src="{{notification.imgURL}}"> {{notification.type.toUpperCase()}} | {{notification.statusMessage}}
+</div>
```
Comment (etterke) [inline Tribes/src/app/notification/notification.component.html]: same thing

File: Tribes/src/app/notification/notification.component.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.container-fluid{
```
Comment (EggDice) [inline Tribes/src/app/notification/notification.component.css]: Missing space after selector

File: Tribes/src/app/notification/notification.component.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    background-color: #263740;
+    border-radius: 20px;
+    padding-top: 10px;
+    color: whitesmoke;
+    list-style-type: none;
+}
+
+img{
```
Comment (EggDice) [inline Tribes/src/app/notification/notification.component.css]: Missing space after selector

**Comment by Erykahh1982** [comment]
made the requested changes

**Pull request merged**